GitHubContents.ps1

```diff
@@ -143,3 +153,266 @@ function Get-GitHubContent
 
     return $result
 }
+
+function Set-GitHubContent
+{
+    <#
+    .SYNOPSIS
+        Sets the contents of a file or directory in a repository on GitHub.
+
+    .DESCRIPTION
+        Sets the contents of a file or directory in a repository on GitHub.
+
+        The Git repo for this module can be found here: http://aka.ms/PowerShellForGitHub
+
+    .PARAMETER OwnerName
+        Owner of the repository.
+        If not supplied here, the DefaultOwnerName configuration property value will be used.
+
+    .PARAMETER RepositoryName
+        Name of the repository.
+        If not supplied here, the DefaultRepositoryName configuration property value will be used.
+
+    .PARAMETER Uri
+        Uri for the repository.
+        The OwnerName and RepositoryName will be extracted from here instead of needing to provide
+        them individually.
+
+    .PARAMETER Path
+        The file path for which to set contents.
+
+    .PARAMETER CommitMessage
+        The Git commit message.
+
+    .PARAMETER Content
+        The new file content.
+
+    .PARAMETER BranchName
+        The branch, or defaults to the default branch of not specified.
+
+    .PARAMETER CommitterName
+        The name of the committer of the commit. Defaults to the name of the authenticated user if
+        not specified. If specified, CommiterEmail must also be specified.
+
+    .PARAMETER CommitterEmail
+        The email of the committer of the commit. Defaults to the email of the authenticated user
+        if not specified. If specified, CommitterName must also be specified.
+
+    .PARAMETER AuthorName
+        The name of the author of the commit. Defaults to the name of the authenticated user if
+        not specified. If specified, AuthorEmail must also be specified.
+
+    .PARAMETER AuthorEmail
+        The email of the author of the commit. Defaults to the email of the authenticated user if
+        not specified. If specified, AuthorName must also be specified.
+
+    .PARAMETER AccessToken
+        If provided, this will be used as the AccessToken for authentication with the
+        REST Api.  Otherwise, will attempt to use the configured value or will run unauthenticated.
+
+    .PARAMETER NoStatus
+        If this switch is specified, long-running commands will run on the main thread
+        with no commandline status update.  When not specified, those commands run in
+        the background, enabling the command prompt to provide status information.
+        If not supplied here, the DefaultNoStatus configuration property value will be used.
+
+    .EXAMPLE
+        Set-GitHubContent  -Path README.md -OwnerName microsoft -RepositoryName PowerShellForGitHub -CommitMessage 'Adding README.md' -Content '# README' -BranchName master
```

Comment:
nit: extra space

```suggestion
        Set-GitHubContent -Path README.md -OwnerName microsoft -RepositoryName PowerShellForGitHub -CommitMessage 'Adding README.md' -Content '# README' -BranchName master
```

Reply:
You'll also likely want to update the parameter ordering too once that gets changed.